tests/distributed/test_basic_distributed_correctness.py

```diff
@@ -41,12 +43,13 @@ def test_models(
     distributed_executor_backend: str,
     attention_backend: str,
     test_suite: str,
+    enable_spmd: bool,
 ) -> None:
 
     if test_suite != TARGET_TEST_SUITE:
         pytest.skip(f"Skip test for {test_suite}")
 
-    if model == "meta-llama/Llama-2-7b-hf" and distributed_executor_backend == "ray" and attention_backend == "" and test_suite == "L4":  # noqa
+    if enable_spmd:
```

Comment:
I still think this change is not necessary. It's better to not surface an argument dedicated to `ray` to the parameter. I prefer the original one, just sparing one test from ray to test adag.

Reply:
Hmm sure. I just thought;
- spmd is not specific to ray. we will make it work with mp.
- this will regress the ray test coverage (one test suite won't run with ray anymore if we replace it to use adag).

Reply:
reverted